pspamm/codegen/architectures/knl/generator.py

```diff
@@ -45,16 +47,15 @@ def make_reg_blocks(self, bm:int, bn:int, bk:int, v_size:int, nnz:int, m:int, n:
         vm = bm//v_size
```

Comment:
See other comments regarding this one. If masking is required to be implemented, it can also be done in a future PR.

Reply:
(cf. `arm/generator.py`)